### PR TITLE
docs: alias links in jsdoc `@see` blocks

### LINKED ISSUE #27844: "Docs: Nuxt config responsiveness is broken."
**Summary (generated)**:

This is a bug report about a line of code in the Nuxt documentation that breaks the reactivity of the Nuxt config page in the API section. The issue is related to the "unhead" module and its installation options. A screenshot is provided to illustrate the problem.

<hr>

### Environment

nuxt.com

### Reproduction

https://nuxt.com/docs/api/nuxt-config#unhead

### Describe the bug

Hi,

This line seems to break reactivity of the nuxt config page in api section in docs:
See: https://unhead.unjs.io/setup/ssr/installation#optionsExample:

here is pic 

Screenshot 2024-06-26 at 15 48 09

### Additional context

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by xjccc**
### 🔗 Linked issue

resolves https://github.com/nuxt/nuxt/issues/27844

### 📚 Description

Alias links

before:

image

after: 

image

### CODE REVIEW

File: packages/schema/src/config/app.ts [hunk 1]
```diff
@@ -314,7 +314,7 @@ export default defineUntypedSchema({
    * @type {string | boolean}
    */
   spaLoadingTemplate: {
-    $resolve: async (val: string | boolean | undefined, get) => typeof val === 'string' ? resolve(await get('srcDir') as string, val) : val ?? null,
+    $resolve: async (val: string | boolean | undefined, get) => typeof val === 'string' ? resolve(await get('srcDir') as string, val) : val ?? false,
```
[the remark this answers is not in the document]
Comment (xjccc) [reply packages/schema/src/config/app.ts:317]: Sorry for that. I will restore it later.

[the remark this answers is not in the document]
Comment (xjccc) [reply packages/schema/src/config/app.ts]: done.😁

File: packages/schema/src/config/app.ts [hunk 2]
```diff
@@ -180,7 +180,7 @@ export default defineUntypedSchema({
      * [enabled in your nuxt.config file](/docs/getting-started/transitions#view-transitions-api-experimental).
      *
      * This can be overridden with `definePageMeta` on an individual page.
-     * @see https://nuxt.com/docs/getting-started/transitions#view-transitions-api-experimental
+     * @see [View Transition API Experimental](https://nuxt.com/docs/getting-started/transitions#view-transitions-api-experimental)
```
Comment (danielroe) [inline packages/schema/src/config/app.ts:183]:
```suggestion
     * @see [Nuxt View Transition API docs](https://nuxt.com/docs/getting-started/transitions#view-transitions-api-experimental)
```

File: packages/schema/src/config/app.ts [hunk 3]
```diff
@@ -168,7 +168,7 @@ export default defineUntypedSchema({
      *
      * This can be overridden with `definePageMeta` on an individual page.
      * Only JSON-serializable values are allowed.
-     * @see https://vuejs.org/api/built-in-components.html#transition
+     * @see [Vue Transition](https://vuejs.org/api/built-in-components.html#transition)
```
Comment (danielroe) [inline packages/schema/src/config/app.ts:171]:
```suggestion
     * @see [Vue Transition docs](https://vuejs.org/api/built-in-components.html#transition)
```

File: packages/schema/src/config/app.ts [hunk 4]
```diff
@@ -158,7 +158,7 @@ export default defineUntypedSchema({
...
```
Comment (danielroe) [inline packages/schema/src/config/app.ts:161]:
```suggestion
     * @see [Vue Transition docs](https://vuejs.org/api/built-in-components.html#transition)
```

**Comment by xjccc** [comment]
I'm sorry it will be late. I will update these change on Monday.😃

**Review by danielroe — APPROVED** [review]
Thank you ❤️

**Pull request merged**